nominees/ottaa-project.json

```diff
@@ -47,11 +53,6 @@
       "org_type": "owner",
       "contact_name": "Hector Costa",
       "contact_email": "hcosta@ottaaproject.com"
-    },
```

Comment:
@nathanbaleeta why are you removing the funder entry? I argue it is still valid and worth keeping it in there.

Reply:
Resolved in commit 1f0e1a1